qt/applications/workbench/workbench/plotting/test/test_figureinteraction.py

```diff
@@ -314,6 +314,14 @@ def test_scripted_plot_show_and_hide_all(self):
         self.interactor.errors_manager.toggle_all_errors(self.ax, make_visible=True)
         self.assertTrue(self.ax.containers[0][2][0].get_visible())
 
+    def test_no_normalisation_options_on_scripted_plot(self):
```

Comment:
This is a nice test. :+1: Can I be uber picky and ask that we rename it to something like `test_no_normalisation_options_on_non_workspace_plot`? A scripted plot from a workspace should be okay, I think, just not one directly from arrays?